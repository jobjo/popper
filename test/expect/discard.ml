open Popper

let test_discarded = test (fun () -> discard)
let () = Clean.run test_discarded
