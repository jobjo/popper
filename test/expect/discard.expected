

PASS: 0/1 tests passed, 1 discarded and 0 failed in XXs.

  ☐  Anonymous  Discarded

